modules/runtime/nuxeo-runtime-migration/src/main/java/org/nuxeo/runtime/migration/MigrationServiceImpl.java

```diff
@@ -194,7 +193,11 @@ protected static KeyValueStore getKeyValueStore() {
     }
 
     public Collection<MigrationDescriptor> getMigrationDescriptors() {
-        return getDescriptors(XP_CONFIG);
+        return getRegistryContributions(XP_CONFIG);
+    }
+
+    protected MigrationDescriptor getMigrationDescriptor(String id) {
+        return (MigrationDescriptor) getRegistryContribution(XP_CONFIG, id).orElse(null);
```

Comment:
Cast needed?

Reply:
Same: orElse brings ambiguity hence the cast